File: src/main/java/org/shark/example/service/cassandra/GetMessageListService.java
package org.shark.example.service.cassandra;

import lombok.RequiredArgsConstructor;
import org.shark.example.datasource.cassandra.MessageRepository;
import org.shark.example.datasource.cassandra.pojo.MessageDo;
import org.springframework.stereotype.Service;

import java.util.List;

@RequiredArgsConstructor
@Service
public class GetMessageListService {

    private final MessageRepository messageRepository;

    public List<MessageDo> getAccountList() {
        return messageRepository.findAll();
    }

}

File: src/main/kotlin/org/shark/example/service/cassandra/GetMessageListService.kt
package org.shark.example.service.cassandra

import org.shark.example.datasource.cassandra.MessageRepository
import org.shark.example.datasource.cassandra.pojo.MessageDo
import org.springframework.stereotype.Service

@Service
class GetMessageListService(
    private val messageRepository: MessageRepository
) {

    fun getAccountList(): List<MessageDo> {
        return messageRepository.findAll()
    }
}
